client/flutter/lib/generated/intl/messages_ar.dart

```diff
@@ -40,7 +40,6 @@ class MessageLookup extends MessageLookupByLibrary {
     "socialDistancing" : MessageLookupByLibrary.simpleMessage(""),
     "throwAwayTissue" : MessageLookupByLibrary.simpleMessage(""),
     "washHands" : MessageLookupByLibrary.simpleMessage(""),
-    "washHandsFrequently" : MessageLookupByLibrary.simpleMessage(""),
```

Comment:
Did you mean to remove this?

Reply:
Yeah, do we need washHands and washHandsFrequently? Also, see above: it was a misunderstanding from another comment haha. I can put it back if needed